Packages/com.unity.inputsystem/InputSystem/Actions/InputActionAsset.cs

```diff
@@ -965,12 +968,14 @@ internal struct WriteFileJsonNoName
         [Serializable]
         internal struct ReadFileJson
         {
+            public int version;
```

Comment:
It's fine to have version here as well. However I believe it should default to version before this PR and not version 0?

Reply:
I realise my previous comment on importer vs json version might have been unjustified since I now realise that this is a file format and somewhat uncorrelated to the importer attribute version. 

If this is introduced I would like us document here inline where we parse JSON what changed with each file format version.